sem/manager.py

```diff
@@ -327,7 +382,10 @@ def run_simulations(self, param_list, show_progress=True, stop_on_errors=True):
 
         # Build ns-3 before running any simulations
         # At this point, we can assume the project was already configured
-        self.runner.configure_and_build(skip_configuration=True)
+        if log_component is not None:
+            self.runner.configure_and_build(optimized=False,skip_configuration=False)    
+        else:
+            self.runner.configure_and_build(skip_configuration=True)
```

Comment:
i stand by my opinion: a user should know about this and act accordingly, but we can discuss about it

Reply:
We'll better discuss this on the next call, I agree